lib/config/rules/typescript.js

```diff
@@ -0,0 +1,41 @@
+// Typescript Specific:
+// These rules find errors related to TypeScript features
+// Reference https://palantir.github.io/tslint/rules/
+
+module.exports = {
+  // Enforces function overloads to be consecutive.
+  'adjacent-overload-signatures': 'off',
+  // Requires explicit visibility declarations for class members. <FEEDBACK>
+  'member-access': 'off',
+  // Enforces member ordering. <FEEDBACK> https://palantir.github.io/tslint/rules/member-ordering/
+  'member-ordering': 'off',
```

Comment:
Created an [issue](https://github.com/Shopify/tslint-plugin-shopify/issues/6) for this. Is it okay if I come back to this after?